chemicaltoolbox/rdkit/sucos_cluster.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="sucos_clustering" name="Cluster ligands using SuCOS" version="0.1">
+    <description>Tool to score docked poses using SuCOS</description>
+    <macros>
+        <import>sucos_macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="2019.03.2.0">rdkit</requirement>
+        <requirement type="package" version="1.3.0">SciPy</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+	    python '$__tool_directory__/sucos_cluster.py' -i '$input' -t $threshold
+    ]]></command>
+    <inputs>
+        <param type="data" name="input" format="sdf" label="The ligands to be clustered" help="Input in SDF format." />
+        <param name="threshold" type="float" value="0.8" min="0" max="1" label="Clustering threshold"/>
+    </inputs>
+    <outputs>
+        <collection name="clusters" type="list" label="Clustered ligands">
+            <!--discover_datasets pattern="__name_and_ext__" ext="sdf"/-->
+            <discover_datasets pattern="(?P&lt;designation&gt;^cluster\d+)\.sdf$" ext="sdf"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" ftype="sdf" value="sucos_cluster.sdf"/>
+            <output_collection name="clusters" type="list">
+                <element name="cluster1" ftype="sdf" file="cluster1.sdf" compare="sim_size" />
+                <element name="cluster2" ftype="sdf" file="cluster2.sdf" compare="sim_size" />
+                <element name="cluster3" ftype="sdf" file="cluster3.sdf" compare="sim_size" />
+                <element name="cluster4" ftype="sdf" file="cluster4.sdf" compare="sim_size" />
+                <element name="cluster5" ftype="sdf" file="cluster5.sdf" compare="sim_size" />
+                <element name="cluster6" ftype="sdf" file="cluster6.sdf" compare="sim_size" />
+            </output_collection>
+        </test>
+    </tests>
+    <help><![CDATA[
+**What it does?**
+
+This tool clusters molecules based on the overlap of 3D features as determined by SuCOS.
+
+Clustering uses a clustering threshold that can be set by the usere. The default threshold is 0.8.s
+
+The original SuCOS code is on GitHub: https://github.com/susanhleung/SuCOS
+The SuCOS work is described here: https://chemrxiv.org/articles/SuCOS_is_Better_than_RMSD_for_Evaluating_Fragment_Elaboration_and_Docking_Poses/8100203
+
+**Input**
+
+Molecules such as an SD file dataset in history.
+
+**Output**
+
+A series of SDF files, one for each cluster containing the molecules in that cluster
+
+    ]]></help>
+    <citations>
+        <citation type="doi">10.26434/chemrxiv.8100203.v1</citation>
```

Comment:
You should use the macro citation that you already have ...

Reply:
done